test/e2e/serial/tests/resource_accounting.go

```diff
@@ -288,65 +294,108 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
-				// so we ensure out pod will not be scheduled on any other node.
-				padPod, err := makeNodePaddingPod(fxt.Namespace.Name, *node, maxResources)
-				if errors.Is(err, e2enrt.ErrNotEnoughResources) {
-					klog.Infof("Node %q has not enough resources without padding", nodeName)
-					continue
+				//calculate base load on the node
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+				By(fmt.Sprintf("computed base load: %s", baseload))
+
+				//get nrt info of the node
+				By(fmt.Sprintf("preparing node %q to fit the test case", nodeName))
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				baseloadRes := corev1.ResourceList{
+					corev1.ResourceCPU:    baseload.CPU,
+					corev1.ResourceMemory: baseload.Memory,
 				}
-				Expect(err).NotTo(HaveOccurred())
 
-				pinnedPadPod, err := pinPodToNode(padPod, nodeName)
-				Expect(err).NotTo(HaveOccurred())
+				// assuming the node's base load is available under the numas, deduct it from any one and fully pad the rest
+				baseloadZone := nrtInfo.Zones[0]
+				By(fmt.Sprintf("selected zone %q to account for the baseload", baseloadZone.Name))
 
-				err = fxt.Client.Create(context.TODO(), pinnedPadPod)
-				Expect(err).NotTo(HaveOccurred())
+				numaCpu, _ := e2enrt.FindResourceAvailableByName(baseloadZone.Resources, corev1.ResourceCPU.String())
+				numaMem, _ := e2enrt.FindResourceAvailableByName(baseloadZone.Resources, corev1.ResourceMemory.String())
+				availableOnNuma := corev1.ResourceList{
+					corev1.ResourceCPU:    numaCpu,
+					corev1.ResourceMemory: numaMem,
+				}
+
+				for resName, resQty := range availableOnNuma {
+					Expect(resQty.Cmp(baseloadRes[resName])).Should(BeNumerically(">=", 0), "resource %q in base load is %s and is greater than the avialable quantity on zone %q, ", resName, baseloadRes[resName], baseloadZone.Name, resQty)
+				}
+
+				//pad the selected zone accounting for the baseload
+				By(fmt.Sprintf("pad zone %q leaving room only for the baseload %s", baseloadZone.Name, e2ereslist.ToString(baseloadRes)))
+
+				//TODO: put this functionality (makePaddingPod, pinPodTo, Create) in one function replacing all the instances
+				padPod, err := makePaddingPod(fxt.Namespace.Name, nrtInfo.Name, baseloadZone, baseloadRes)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+				padPod, err = pinPodTo(padPod, nrtInfo.Name, baseloadZone.Name)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "unable to pin pod %q to zone %q", padPod.Name, baseloadZone.Name)
 
-				paddingPods = append(paddingPods, pinnedPadPod)
+				err = fxt.Client.Create(context.TODO(), padPod)
+				Expect(err).ToNot(HaveOccurred())
+				paddingPods = append(paddingPods, padPod)
+
+				//pad each zone with a pod so the test would be valid for both TM scopes: container & pod levels
```

Comment:
let's focus on `scope=pod` now and let's make changes all across the suite to support `scope=container` later when it's available.
Also, I fail to see the benefit of the code here (lines 369-389)

Reply:
readjusted the flow a bit, may you have a look when possible? thanks!